trapping-rainwater/main.c

```diff
@@ -19,6 +19,43 @@ int maxTrapValue(int *height, int heightSize, int pos)
     return maxTrap;
 }
 
+int min(int a,int b){
+    if(a<b)
+    return a;
+    return b;
+}
+int max(int a,int b){
+    if(a>b)
+    return a;
+    return b;
+}
+//Trapping rainwater in O(n)
+int trappingRainWaterAakash(int *height, int heightSize){
+    int trap = 0;
+    int maxHeightLeft[heightSize],maxHeightRight[heightSize];
+    for(int i=0;i<heightSize;i++){
+        maxHeightLeft[i]=0;
+        maxHeightRight[i]=0;
+    }
```

Comment:
We can improve here a bit by using `memset` instead of a loop:

```suggestion
    int maxHeightLeft[heightSize], maxHeightRight[heightSize];
    memset(maxHeightLeft, 0, heightSize);
    memset(maxHeightRight, 0, heightSize);
```

Reply:
So we can use memset to initialise the array to any number. That's awesome